include/rdma/fi_ext.h

```diff
@@ -123,6 +124,49 @@ struct fid_mem_monitor {
 	struct fi_ops_mem_notify *import_ops;
 };
 
+/*
+ * System logging import extension:
+ * To use, open logging fid and import.
+ */
+
+struct fi_ops_log {
+	size_t size;
+	int (*enabled)(int disable, enum fi_log_level level,
+		       enum fi_log_subsys subsys);
+	int (*ready)(int disable, enum fi_log_level level,
+		     enum fi_log_subsys subsys, uint64_t *showtime);
+	void (*log)(const char *buf);
```

Comment:
I finally decided not to use `const char *fmt, va_list` as input because I didn't see any added value for the application.
I think a simple char *buffer makes more sense here.